buildenv/jenkins/docker-slaves/ppc64le/ubuntu16/Dockerfile

```diff
@@ -0,0 +1,101 @@
+# Copyright (c) 2018, 2018 IBM Corp. and others
+#
+# This program and the accompanying materials are made available under
+# the terms of the Eclipse Public License 2.0 which accompanies this
+# distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+# or the Apache License, Version 2.0 which accompanies this distribution and
+# is available at https://www.apache.org/licenses/LICENSE-2.0.
+#
+# This Source Code may also be made available under the following
+# Secondary Licenses when the conditions for such availability set
+# forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+# General Public License, version 2 with the GNU Classpath
+# Exception [1] and GNU General Public License, version 2 with the
+# OpenJDK Assembly Exception [2].
+#
+# [1] https://www.gnu.org/software/classpath/license.html
+# [2] http://openjdk.java.net/legal/assembly-exception.html
+#
+# SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+
+# To use this docker file:
+# First copy your public ssh key into a file named authorized_keys next to the Dockerfile
+# Then include a known_hosts file next to the Dockerfile, with github as a saved host
+# This can be done with "ssh-keyscan github.com >> path_to_dockerfile/known_hosts"
+# Make sure you are in the directory contianing the Dockerfile and authorized_keys file
+# Then run:
+#   docker build -t openj9 -f Dockerfile .
+#   docker run -it openj9
+
+FROM ubuntu:16.04
+
+# Install required OS tools
+
+ENV USER="jenkins"
+
+RUN apt-get update \
+  && apt-get install -qq -y --no-install-recommends \
+    software-properties-common \
+    python-software-properties \
+  && add-apt-repository ppa:ubuntu-toolchain-r/test \
+  && apt-get update \
+  && apt-get install -qq -y --no-install-recommends \
+    ant \
+    ant-contrib \
+    autoconf \
+    build-essential \
+    cmake \
+    cpio \
+    curl \
+    file \
+    g++-4.8 \
+    gcc-4.8 \
+    git \
+    git-core \
```

Comment:
Not sure about this one?
https://askubuntu.com/questions/5930/what-are-the-differences-between-the-git-and-git-core-packages